README.md

```diff
@@ -1,4 +1,13 @@
-# Kids First Sphinx Docs Theme
+<p align="center">
+  <img src="docs/images/logo.png">
```

Comment:
Can you put an alt tag in here with a `width=660px`?

Reply:
👍